syntax = "proto3";

service OrderService {
  rpc GetOrder(OrderId)returns(Order){}
}


message OrderId{
  int32 Id = 1;
}

message Order {
  int32 UserId = 1;
  int32 ProductId = 2;
  int32 Count = 3;
  string Name = 4;
}